index.src.html

```diff
@@ -4133,6 +4136,43 @@
 : <dfn event for=ImageTrack>change</dfn>
 :: Fired at the {{ImageTrack}} when the {{ImageTrack/frameCount}} is altered.
 
+Resource Reclamation{#resource-reclamation}
+==============================================
+
+When resources are constrained, a User Agent may proactively reclaim codecs.
+This is particularly true in the case where hardware codecs are limited, and
+shared accross all web pages.
+
+A User Agent may only [=reclaim a codec=] that is either an
+[=inactive codec=], a [=background codec=], or both. A User Agent MUST NOT
+reclaim a codec that is both active and in the foreground, e.g any codec that is
+neither an [=inactive codec=] nor a [=background codec=].
+
+An <dfn>inactive codec</dfn> is a codec that has not received a call to
+`encode()` or `decode()` in the past `3 seconds`, and has not called its
+`output()` callback in the past `3 seconds`.
+
+A <dfn>background codec</dfn> is a codec whose {{ownerDocument}}'s
```

Comment:
Does {{ownerDocument}} also work for the workers case?

Reply:
You can refer to the document that is in the owner set of the worker if there is one, or reccurse if there is none (nested workers).